### PR TITLE
Fix when nested DTO have casted field

### PR DESCRIPTION
**Pull request by bastien-phi**
Not yet sure how to fix this but it looks like I found a little issue.

I will dig into it but feel free to add the fix to this PR !

### CODE REVIEW

**Comment by bastien-phi** [comment]
Looks like I found a fix.

Removing the `dd()` is required here (not sure why it was here BTW)

File: src/Reflection/DataTransferObjectProperty.php [hunk 1]
```diff
@@ -125,6 +126,6 @@ private function resolveCasterFromDefaults(): ?Caster
             }
         }
 
-        dd($defaultCastAttributes);
+        return null;
```
Comment (mfn) [inline src/Reflection/DataTransferObjectProperty.php:129]: 😄

**Comment by bastien-phi** [comment]
Hi @brendt !

Sorry for pinging you. Is there something wrong with this PR ? I hope this is the correct way to fix the issue !

**Comment by brendt** [comment]
> not sure why it was here BTW 

That was my bad, thanks for removing it. Looks good, thanks for fixing!

**Comment by brendt** [comment]
Released as 3.0.3: https://github.com/spatie/data-transfer-object/releases/tag/3.0.3

**Comment by bastien-phi** [comment]
Thanks for merging !

**Pull request merged**